x/evm/keeper/state_transition_test.go

```diff
@@ -27,18 +27,13 @@ import (
 	"github.com/evmos/evmos/v18/testutil/integration/evmos/utils"
 	utiltx "github.com/evmos/evmos/v18/testutil/tx"
 	"github.com/evmos/evmos/v18/x/evm/keeper"
-	"github.com/evmos/evmos/v18/x/evm/statedb"
 	"github.com/evmos/evmos/v18/x/evm/types"
+	evmtypes "github.com/evmos/evmos/v18/x/evm/types"
 	feemarkettypes "github.com/evmos/evmos/v18/x/feemarket/types"
 )
 
-func (suite *EvmKeeperTestSuite) TestGetHashFn() {
-	keyring := testkeyring.New(1)
-	unitNetwork := network.NewUnitTestNetwork(
-		network.WithPreFundedAccounts(keyring.GetAllAccAddrs()...),
-	)
-
-	header := unitNetwork.GetContext().BlockHeader()
+func (suite *KeeperTestSuite) TestGetHashFn() {
+	header := suite.network.GetContext().BlockHeader()
```

Comment:
In the previous version the network was started from scratch in every test, here it seems that all tests are using the same network. Am I wrong?

Reply:
yeah, you are right. Added a reset in each test